spec/test-data/translations/glotpress-download/expected/values-fr/strings.xml

```diff
@@ -24,7 +23,7 @@ Language: fr
     <string name="shipping_label_create_price_total">Total de la commande</string>
     <string name="shipping_label_create_price_woo_discount_content_description">En savoir plus sur la remise WooCommerce Services</string>
     <string name="shipping_label_create_price_woo_discount">Remise WooCommerce Services</string>
-    <string name="shipping_label_create_price_subtotal">Sous-total</string>
+    <string name="shipping_label_create_price_subtotal">Sous&amp;#8211;total</string>
```

Comment:
This result is incorrect for 2 reasons:

1. This will lead to the text being `Sous&#8211total` because the `&amp;` will be replaced by `&` but not escape the `#8211` hex code.
2. This is typically an example of context where the regular dash (aka hyphen) is the correct use, not the en dash.

The short version is that `en dash` is for ranges or rhetorical pause, while hyphen is for compound words (like here).